tasklist/client/src/Tasks/CollapsiblePanel/index.tsx

```diff
@@ -88,35 +88,58 @@ const CollapsiblePanel: React.FC = () => {
 
   if (isCollapsed) {
     return (
-      <div className={cn(styles.base, styles.collapsedContainer)}>
-        <Button
-          hasIconOnly
-          iconDescription="Expand to show filters"
-          tooltipPosition="right"
-          onClick={() => {
-            setIsCollapsed(false);
-          }}
-          renderIcon={SidePanelOpen}
-          size="md"
-          kind="ghost"
-        />
-        <Button
-          hasIconOnly
-          iconDescription="Custom filter"
-          tooltipPosition="right"
-          onClick={() => {}}
-          renderIcon={Filter}
-          size="md"
-          kind="ghost"
-        />
-      </div>
+      <nav
+        id="task-nav-bar"
+        aria-labelledby="filters-title"
```

Comment:
I'd remove this. It doesn't refer to anything

Reply:
yep, it was a leftover from something else I was trying